algoliasearch-core/src/test/java/com/algolia/search/integration/TestHelpers.java

```diff
@@ -5,6 +5,15 @@
 import java.time.format.DateTimeFormatter;
 
 public class TestHelpers {
+
+  public static String ALGOLIA_APPLICATION_ID_1 = System.getenv("ALGOLIA_APPLICATION_ID_1");
+  public static String ALGOLIA_API_KEY_1 = System.getenv("ALGOLIA_ADMIN_KEY_1");
+  public static String ALGOLIA_SEARCH_KEY_1 = System.getenv("ALGOLIA_SEARCH_KEY_1");
+  public static String ALGOLIA_APPLICATION_ID_2 = System.getenv("ALGOLIA_APPLICATION_ID_2");
+  public static String ALGOLIA_API_KEY_2 = System.getenv("ALGOLIA_ADMIN_KEY_2");
+  public static String ALGOLIA_APPLICATION_ID_MCM = System.getenv("ALGOLIA_APPLICATION_ID_MCM");
+  public static String ALGOLIA_ADMIN_KEY_MCM = System.getenv("ALGOLIA_ADMIN_KEY_MCM");
```

Comment:
I'd suggest renaming all the variables with the same name of the environment variable they are bound to. This would lead to less confusing and back-and-forth when debugging from my own experience. (`Shift+F6` to rename everywhere in IntelliJ ;) )

Reply:
You're right, let's stay consistent. I'll do it.